spec/system/initiative_spec.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Initiative", type: :system do
+  let(:organization) { create(:organization) }
+  let(:state) { :published }
+  let!(:user) { create(:user, :confirmed, organization: organization) }
+  let(:base_initiative) do
+    create(:initiative, organization: organization, state: state)
+  end
+
+  before do
+    switch_to_host(organization.host)
+    login_as(user, scope: :user)
+  end
+
+  context "when the initiative does not exist" do
+    it_behaves_like "a 404 page" do
+      let(:target_path) { decidim_initiatives.initiative_path(99_999_999) }
+    end
+  end
+
+  describe "initiative page" do
+    let!(:initiative) { base_initiative }
+    let(:attached_to) { initiative }
+
+    context "when requesting the initiative path" do
+      before do
+        visit decidim_initiatives.initiative_path(initiative)
+      end
+
+      shared_examples_for "initiative shows signatures" do
+        it "shows signatures for the state" do
+          expect(page).to have_css(".progress__bar__number")
+          expect(page).to have_css(".progress__bar__text")
+        end
+      end
+
+      shared_examples_for "initiative does not show signatures" do
+        it "does not show signatures for the state" do
+          expect(page).not_to have_css(".progress__bar__number")
+          expect(page).not_to have_css(".progress__bar__text")
+        end
+      end
+
+      shared_examples_for "initiative shows recepisse link" do
+        it "shows recepisse link" do
+          expect(page).to have_link("Voir le récépissé")
```

Comment:
Default locale seems to be `en`, I think we should find the english translation for print link